.github/workflows/build_mac.yml

```diff
@@ -27,16 +27,15 @@ jobs:
     - name: Install Python
       run: brew install python@3.12 --overwrite
 
+    - name: Configure dirmgr
```

Comment:
is this intentionally added?

Reply:
Yes, as per the original discussion In [actions/runner-images#9777](https://github.com/actions/runner-images/issues/9777), this is required, in addition to the runner now being fixed, one doesn't work without the other it seems.